apps/explorer/package.json

```diff
@@ -13,48 +13,48 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^15.0.0",
-    "@angular/common": "^15.0.0",
-    "@angular/compiler": "^15.0.0",
-    "@angular/core": "^15.0.0",
-    "@angular/forms": "^15.0.0",
-    "@angular/platform-browser": "^15.0.0",
-    "@angular/platform-browser-dynamic": "^15.0.0",
-    "@angular/router": "^15.0.0",
-    "@apollo/client": "^3.7.3",
-    "apollo-angular": "^4.1.1",
-    "graphql": "^16.6.0",
-    "rxjs": "~7.5.0",
-    "tslib": "^2.3.0",
-    "zone.js": "~0.12.0"
+    "@angular/animations": "15.2.0",
+    "@angular/common": "15.2.0",
+    "@angular/compiler": "15.2.0",
+    "@angular/core": "15.2.0",
+    "@angular/forms": "15.2.0",
+    "@angular/platform-browser": "15.2.0",
+    "@angular/platform-browser-dynamic": "15.2.0",
+    "@angular/router": "15.2.0",
+    "@apollo/client": "3.7.9",
+    "apollo-angular": "4.2.1",
+    "graphql": "16.6.0",
+    "rxjs": "7.8.0",
+    "tslib": "2.5.0",
+    "zone.js": "0.12.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^15.0.3",
-    "@angular/cli": "~15.0.3",
-    "@angular/compiler-cli": "^15.0.0",
-    "@angular/google-maps": "^15.0.3",
-    "@graphql-codegen/cli": "^2.16.2",
-    "@graphql-codegen/typescript": "^2.8.6",
+    "@angular-devkit/build-angular": "15.2.0",
+    "@angular/cli": "15.2.0",
+    "@angular/compiler-cli": "15.2.0",
+    "@angular/google-maps": "15.2.0",
+    "@graphql-codegen/cli": "3.2.0",
+    "@graphql-codegen/typescript": "3.0.1",
     "@graphql-codegen/typescript-apollo-angular": "^3.5.6",
-    "@graphql-codegen/typescript-operations": "^2.5.11",
+    "@graphql-codegen/typescript-operations": "3.0.1",
     "@sni/prettier-config": "*",
-    "@types/geojson": "^7946.0.10",
-    "@types/jasmine": "~4.3.0",
-    "date-fns": "^2.29.3",
+    "@types/geojson": "7946.0.10",
+    "@types/jasmine": "4.3.1",
+    "date-fns": "2.29.3",
     "eslint-config-sni": "*",
-    "jasmine-core": "~4.5.0",
-    "karma": "~6.4.0",
-    "karma-chrome-launcher": "~3.1.0",
-    "karma-coverage": "~2.2.0",
-    "karma-jasmine": "~5.1.0",
-    "karma-jasmine-html-reporter": "~2.0.0",
-    "ngx-date-fns": "^10.0.1",
-    "typescript": "4.8.4",
-    "wkt-parser-helper": "^4.1.0"
+    "jasmine-core": "4.5.0",
+    "karma": "6.4.1",
+    "karma-chrome-launcher": "3.1.1",
+    "karma-coverage": "2.2.0",
+    "karma-jasmine": "5.1.0",
+    "karma-jasmine-html-reporter": "2.0.0",
+    "ngx-date-fns": "10.0.1",
+    "typescript": "4.9.5",
+    "wkt-parser-helper": "4.1.0"
   },
   "overrides": {
     "apollo-angular": {
-      "@angular/core": "^15.0.0"
+      "@angular/core": "15.0.0"
```

Comment:
@giovanneguerra Are you sure we need this override?

Reply:
I'm not sure, I'll research and get back to you.

> @giovanneguerra Are you sure we need this override?

Reply:
@smakhtin removed overrides